src/System.Net.Http/src/System/Net/Http/MultipartContent.cs

```diff
@@ -34,19 +34,26 @@ public class MultipartContent : HttpContent, IEnumerable<HttpContent>
         #region Construction
 
         public MultipartContent()
-            : this("mixed", GetDefaultBoundary())
+            : this("mixed", GetDefaultBoundary(), "\r\n")
         { }
 
         public MultipartContent(string subtype)
-            : this(subtype, GetDefaultBoundary())
+            : this(subtype, GetDefaultBoundary(), "\r\n")
         { }
-
+        
         public MultipartContent(string subtype, string boundary)
+            : this(subtype, GetDefaultBoundary, "\r\n")
+        { }
+        
+        public MultipartContent(string subtype, string boundary, string crlf)
```

Comment:
To make it a constructive complain, please suggest a nice way to let the user of the MultipartContent choose the linefeed he needs.

You are right about the hardcoded constant, also private string crlf; could be made read only.

If there is any workaround to this issue I would also appreciate.

Thanks